src/Atmos/Model/linear.jl

```diff
@@ -0,0 +1,68 @@
+abstract type AtmosLinearModel <: BalanceLaw
+end
+
+
+vars_state(lm::AtmosLinearModel, T) = vars_state(lm.atmos,T)
+vars_gradient(lm::AtmosLinearModel, T) = @vars()
+vars_diffusive(lm::AtmosLinearModel, T) = @vars()
+vars_aux(lm::AtmosLinearModel, T) = vars_aux(lm.atmos,T)
+vars_integrals(lm::AtmosLinearModel,T) = @vars()
+
+
+update_aux!(dg::DGModel, lm::AtmosLinearModel, Q::MPIStateArray, auxstate::MPIStateArray, t::Real) = nothing
+integrate_aux!(lm::AtmosLinearModel, integ::Vars, state::Vars, aux::Vars) = nothing
+flux_diffusive!(lm::AtmosLinearModel, flux::Grad, state::Vars, diffusive::Vars, aux::Vars, t::Real) = nothing
+function wavespeed(lm::AtmosLinearModel, nM, state::Vars, aux::Vars, t::Real)
+  ref = aux.ref_state
+  return soundspeed_air(ref.T)
+end
+
+function boundary_state!(nf::Rusanov, lm::AtmosLinearModel, x...)
+  atmos_boundary_state!(nf, NoFluxBC(), lm.atmos, x...)
+end
+function boundary_state!(nf::CentralNumericalFluxDiffusive, lm::AtmosLinearModel, x...)
+  nothing
+end
```

Comment:
@jkozdon @mwarusz does this seem a reasonable default?

Reply:
Seems reasonable to me.

Reply:
I agree, seems fine.